File: runelite-client/src/main/java/net/runelite/client/plugins/microbot/vorkath/CROSSBOW.java
package net.runelite.client.plugins.microbot.vorkath;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor
public enum CROSSBOW {
    DRAGON_HUNTER_CROSSBOW("Dragon hunter crossbow"),
    RUNE_CROSSBOW("Rune crossbow"),
    DRAGON_CROSSBOW("Dragon crossbow");

    private final String crossbowName;

    @Override
    public String toString() { return crossbowName; }
}

File: runelite-client/src/main/java/net/runelite/client/plugins/jrPlugins/autoVorkath/enums/CROSSBOW.kt
/*
 * Copyright (c) 2024. By Jrod7938
 *
 */
package net.runelite.client.plugins.jrPlugins.autoVorkath.enums

import lombok.Getter
import lombok.RequiredArgsConstructor

@Getter
@RequiredArgsConstructor
enum class CROSSBOW(private val crossbowName: String) {
    DRAGON_HUNTER_CROSSBOW("Dragon hunter crossbow"),
    RUNE_CROSSBOW("Rune crossbow"),
    ARMADYL_CROSSBOW("Armadyl crossbow"),
    DRAGON_CROSSBOW("Dragon crossbow");

    override fun toString(): String {
        return crossbowName
    }
}
